boards/arm/xenvm/xenvm.dts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2020 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * This file was created by running
+ * # LIBXL_DEBUG_DUMP_DTB=domu-libxl.dtb xl create zephyr.conf
+ * decompilling resulting domu-libxl.dtb and then munally aligning it
+ * with zephyr requirements.
+ */
+
+/dts-v1/;
+
+#include <arm/armv8-a.dtsi>
+#include <dt-bindings/interrupt-controller/arm-gic.h>
+
+/ {
+	model = "XENVM-4.15";
+	compatible = "xen,xenvm-4.15\0xen,xenvm";
+	interrupt-parent = <&gic>;
+	#address-cells = <0x02>;
+	#size-cells = <0x02>;
+
+	chosen {
+		zephyr,console = &sbsa;
+		zephyr,shell-uart = &sbsa;
+	};
+
+	cpus {
+		#address-cells = <0x01>;
+		#size-cells = <0x00>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,armv8";
+			enable-method = "psci";
+			reg = <0x00>;
+		};
+	};
+
+	psci {
+		compatible = "arm,psci-1.0\0arm,psci-0.2\0arm,psci";
+		method = "hvc";
+		cpu_off = <0x01>;
+		cpu_on = <0x02>;
+	};
+
+	memory@40000000 {
+		device_type = "memory";
+		reg = <0x00 0x40000000 0x00 0x10000000>;
+	};
+
+	gic: interrupt-controller@3001000 {
+		compatible = "arm,cortex-a15-gic", "arm,cortex-a9-gic", "arm,gic";
```

Comment:
What if a user want to run Zephyr with GICv3? Would we need to create a new Device-Tree?

Reply:
Yes. Zephyr does not parses device tree at runtime. During compilation, special scripts generates header file from a provided DTS, with all needed definitions.

Reply:
I added board manual. It describes how to update Zephyr configuration in case of any changes from Xen side.